keyboards/atlantis/ak81_ve/info.json

```
@@ -0,0 +1,9 @@
+{
+    "keyboard_name": "AK81_VE", 
+    "maintainer": "fOmey", 
+    "layouts": {
+        "LAYOUT_ansi": {
```

The name used here must be an exact match for the corresponding macro in `ak81_ve.h`:

https://github.com/qmk/qmk_firmware/blob/2c28f8e04cb202311b0c4c763da685be3c316d84/keyboards/atlantis/ak81_ve/ak81_ve.h#L21

```suggestion
        "LAYOUT": {
```